galata/test/jupyterlab/texteditor.test.ts

```diff
@@ -57,10 +57,12 @@ test.describe('Text Editor Tests', () => {
 
     await page.locator(`[role="main"] >> text=${DEFAULT_NAME}`).waitFor();
 
-    await page.type(
-      '.cm-content',
-      'Not active\nActive line with >>selected text<<\nNot active'
-    );
+    await page.pause();
+    await page
+      .locator('.jp-FileEditorCodeWrapper .cm-content')
+      .pressSequentially(
+        'Not active\nActive line with >>selected text<<\nNot active'
+      );
```

Comment:
I would expect `fill` to work here. Does it not?

Reply:
It should, yes

Reply:
I updated it